tests/fixture-helper.ts

```diff
@@ -0,0 +1,30 @@
+import { matchesFixture } from './assertions.js';
+import { copyFixture, runScript } from './utils.js';
+
+export class AddonFixtureHelper {
```

Comment:
I don't want to overcomplicate things, but with #151 providing some testing infrastructure, should't this file and `copyFixture` ideally been part of that PR rather? 😅

Another though here, and I am totally fine ignoring this for now, like doing later or not doing at all, but given we have a more elaborate testing class here (i.e. it own some state), does it make sense to integrate `createAddon` into this? Like rename to e.g. `TestAddonBuilder` and do the addon creation as part of the constructor? Just thinking aloud here...

Reply:
> should't this file and copyFixture ideally been part of that PR rather? 

yes! I will move it over there -- I have a fear of making PRs too big -- especially when utils-prs really need usages to see how the utils will be used / what their purpose is.


> does it make sense to integrate createAddon into this? Like rename to e.g. TestAddonBuilder and do the addon creation as part of the constructor? 

I really like this idea :tada: will do